src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -325,9 +325,26 @@ protected Map<String, Object> create(final SscsCaseDataWrapper responseWrapper,
         personalisation.put(PARTY_TYPE, subscriptionWithType.getParty().getClass().getSimpleName());
         personalisation.put(ENTITY_TYPE, subscriptionWithType.getEntity().getClass().getSimpleName());
 
+        personalisation.put(DOCUMENT_TYPE_NAME, getDocumentType(notificationEventType));
+        personalisation.put(SENDER_NAME, LetterUtils.getNameForSender(ccdResponse));
+
         return personalisation;
     }
 
+    private static String getDocumentType(NotificationEventType eventType) {
```

Comment:
Would this be better named as getPostHearingDocumentType? I can't really tell from the context, but if the types are always going to be post hearing then I think that name would be more clear.

Reply:
Renamed method and moved "action further evidence" personalisation into separate class